server/interceptor/interceptor.go

```diff
@@ -128,37 +128,28 @@ type trillianProcessor struct {
 }
 
 func (tp *trillianProcessor) Before(ctx context.Context, req interface{}) (context.Context, error) {
-	defer func() {
-		var treeID int64
-		if tp.info != nil {
-			treeID = tp.info.treeID
-		}
-		requestCounter.Inc(fmt.Sprint(treeID))
-	}()
-
 	quotaUser := tp.parent.qm.GetUser(ctx, req)
-	info, err := getRPCInfo(req, quotaUser)
+	info, err := newRPCInfo(req, quotaUser)
 	if err != nil {
 		incRequestDeniedCounter(badInfoReason, 0, quotaUser)
```

Comment:
(pre-existing) should we `glog.Warningf` these, given that they indicate an internal error?  (Is there a Go `LOG_EVERY_N`?)

Reply:
Sounds good. Added a warning to "badInfo" errors, but not to GetTree errors, are there are many "legitimate" that arise from it (not found, frozen / deleted trees, etc)